File: engine/model/src/main/scala/tools/packet/character/SetAutoHpPot.scala
package tools.packet.character

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class SetAutoHpPot(private var _itemId: Int) extends PacketInput {
  def itemId: Int = _itemId

  override def opcode(): SendOpcode = SendOpcode.AUTO_HP_POT
}

File: engine/engine-model/src/main/java/tools/packet/character/SetAutoHpPot.java
package tools.packet.character;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record SetAutoHpPot(Integer itemId) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.AUTO_HP_POT;
   }
}